src/main/kotlin/com/game/catbackend/api/dto/JoinLobbyDTO.kt

```diff
@@ -0,0 +1,6 @@
+package com.game.catbackend.api.dto
+
+
+class JoinLobbyDTO(
+    val userName: String
```

Comment:
Adicionar anotation para verificar se o username foi preenchido 
```suggestion
   @Size(min = 1, max = 50, message = "Username must be filled")
    val userName: String
```